### ISSUE TITLE
Zoom sometimes opens in tiling mode despite system exception

### ISSUE THREAD

**Issue by dawaltconley** [pop-os/shell#1442]
**(1) Issue/Bug Description:**

Zoom sometimes tiles its windows despite the system exception for zoom windows. Either this just started with a recent Zoom update, or I somehow haven't noticed til now.

**(2) Steps to reproduce (if you know):**

This occurs for me when, 
1. Zoom is not yet open or running in the background
2. Window tiling is enabled
3. Click on a zoom link in Firefox, prompting zoom to open

At least Zoom version 5.10.4 (2845)

**(3) Expected behavior:**

Zoom windows should not tile, as when the app is launched from the dock or command prompt.

**(4) Distribution (run `cat /etc/os-release`):**
 
Pop!_OS 22.04 LTS

**(5) Gnome Shell version:**

GNOME Shell 42.0

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```
pop-shell:
  Installed: 1.1.0~1652463555~22.04~034fb14
  Candidate: 1.1.0~1652463555~22.04~034fb14
  Version table:
 *** 1.1.0~1652463555~22.04~034fb14 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1001 http://apt.pop-os.org/release jammy/main i386 Packages
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**

Upgraded from 20.10, originally installed from System76 website.

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**

 1 x 2560 x 1440

**(9) Other Installed/Enabled Extensions:**

```
bluetooth-quick-connect@bjarosze.gmail.com
ControlBlurEffectOnLockScreen@pratap.fastmail.fm
ding@rastersoft.com
espresso@coadmunkee.github.com
freon@UshakovVasilii_Github.yahoo.com
gsconnect@andyholmes.github.io
hide-universal-access@akiirui.github.io
openweather-extension@jenslody.de
unblank@sun.wxg@gmail.com
windowIsReady_Remover@nunofarruca@gmail.com
```

**Comment by n3m0-22** [comment]
I can confirm this is happening. Tested with and without the extensions listed as well as [#1439](https://github.com/pop-os/shell/pull/1439).

**Comment by dtorres-sf** [comment]
This was happening to me. When I checked out the window title using `xprop | grep WM_CLASS` I got a very long string such as `join?action=join&confno=111111111&pwd=OPOCXXcxTX....`. I then followed these steps:

1. Signed into zoom (for me it was a fresh install and I was signed out)
2. Exited Zoom
3. Re-opened Zoom
4. Zoom app was now floating.
5. I checked the windows class again via `xprop | grep WM_CLASS` and saw it was `WM_CLASS(STRING) = "zoom ", "zoom "`

Worth noting for me this started for me when I updated to zoom `Version: 5.10.4 (2845)`
